docs/source/model_design/concept_model/standard_results_formats/index.rst

```diff
@@ -0,0 +1,197 @@
+  Section Level 3
+  ~~~~~~~~~~~~~~~
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _vivarium_best_practices_standard_results_format:
+
+=========================================================
+Vivarium Standard Results Format
+=========================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+The aim of this page is to outline a standard format for Vivarium model results as they appear in the count data that the engineers pass off to the researchers.
+
+Motivation behind standard formatting
+-------------------------------------
+
+Standardized formatting of Vivarium model results may be beneficial in the following ways:
+
+- Predictability of results format allows researchers to write results processing code in advance of receiving results
+
+- Results format consistency enables generalizable results processing code (for researchers and engineers) and reduces the need to code exception cases
+
+- Shared language and classification of terms may aid in communication between researchers and engineers
+
+Formatting guide
+-----------------
+
+General considerations
+++++++++++++++++++++++
+
+1. Consistency with GBD names is preferred over internal/other types of consistency. Generally, GBD names (such as age_group_name, location_name, cause_name, etc.) should appear in Vivarium results as :code:`{gbd_name}.replace(" ","_").lower()`. Note that sometimes GBD names things in inconsistent ways. For instance, "6-11 months" and "12 to 23 months". However, this will allow us to easily merge Vivarium results and GBD data using their shared names.
+
+2. Successful execution of these standard formats depends on researcher documentation of requested model outputs within the simulation concept model document.
+
+.. todo::
+
+  Link to updated template doc when ready
+
+3. If a researcher-requested output does not clearly fit within the examples outlined below, researchers and engineers should meet to determine desired results formatting and document the format within the project concept model requested outputs section.
+
+Parameters in results dataframes
+++++++++++++++++++++++++++++++++
+
+.. list-table:: Parameters
+  :header-rows: 1
+
+  * - Parameter
+    - Value type
+    - Definition
+    - Possible values
+  * - :code:`input_draw`
+    - integer
+    - Simulated input draw number
+    - [0, 999]
+  * - :code:`scenario`
+    - string
+    - Simulated scenario name
+    - * baseline
+      * alternative
+      * etc.
+  * - :code:`age`
+    - string
+    - Age group of stratum
+    - :code:`gbd_age_group_name.replace(" ","_").lower()`
```

Comment:
Great point! I updated the age and year columns to include instructions here. For sex I just added another option of "both", which is the same as the GBD name for sex_id=3.

Reply:
I figured it was better to actually define age/year start/end instead of "all" just to make sure we know what it represents.

Reply:
That makes sense! But, is it always/usually the case that we have "simulation minimum and **maximum** age values"? I was under the impression that for most of our open cohort sims, we had minimum 0 with no set maximum. In that case, I think it would make sense to consider "All Ages" -> "all_ages" (age_group ID 22) the matching GBD age group, since that is the GBD result we would want to compare with if results were not age-stratified.

Reply:
Another good point. I imagine we could have a sim with "adults" that would be something like "20_to_95_plus", so I'll add a note about what to do if there is no age max and then also add a note about all_ages being preferred over 0_to_95_plus. Let me know if you think this doesn't cover it, though!

Reply:
Oops, coming back to say that we should do "{age_min}_plus" when there is no age max, because that is more consistent with GBD.